torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14443,8 +14443,10 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                          dtypes=(torch.bfloat16, torch.float16), active_if=PLATFORM_SUPPORTS_FUSED_ATTENTION),
             DecorateInfo(unittest.expectedFailure, "TestMeta", "test_dispatch_symbolic_meta_outplace",
                          dtypes=(torch.float32,), active_if=PLATFORM_SUPPORTS_MEM_EFF_ATTENTION),
-            DecorateInfo(unittest.expectedFailure, "TestMeta", "test_dispatch_meta_outplace_all_strides"),
```

Comment:
we don't need test_dispatch_meta_outplace_all_strides anymore?

Reply:
this test doesn't actually exist :)